src/parser/jobs/blm/modules/Gauge.js

```diff
@@ -104,21 +104,20 @@ export default class Gauge extends Module {
 
 			this.updateStackTimers(event)
 
-			if (!this.parser.byPlayer(event) || !GAUGE_EVENTS.includes(event.type)) {
-				continue
+			if (!GAUGE_EVENTS.includes(event.type)) { continue }
+			if (this.parser.byPlayer(event)) {
+				switch (event.type) {
```

Comment:
For the sake of fixing the bug I'll let it pass, but this is a bit ugly.

You can do stuff like

```js
switch (event.type) {
case 'cast':
	this._onCast(event)
	// fall through
case 'begincast':
case 'damage':
	break;
}
```